### ISSUE TITLE
Shutdown and restart icons too big in XFCE

### ISSUE THREAD

**Issue by andrewhotlab** [zayronxio/Zafiro-icons#93]
Thank you so much for the great work with this icon theme!

In the latest release i noticed this "problem" with XFCE:

![2021-11-04_18-20](https://user-images.githubusercontent.com/26433166/140388397-37d54b9b-d89c-4a79-9777-2a09a43d1533.png)

![2021-11-04_18-23](https://user-images.githubusercontent.com/26433166/140388698-f493c158-661f-4ae9-86c5-56d8962e063a.png)

I can't tell you when exactly, but a few months ago the same icon was appearing smaller in these dialog windows.

**Comment by tio-trom** [comment]
Agree. I would also recommend to make these buttons black and white, to differentiate between apps and system commands.
